Dockerfile

```
@@ -12,6 +12,7 @@ WORKDIR /app
 # Set production environment
 ENV NODE_ENV="production"
 ENV SENTRY_ENVIRONMENT="production"
+ENV REQUEST_LOGGING false
```

```suggestion
ENV REQUEST_LOGGING="false"
```

Are booleans supported here?

---

It's all strings. I copied this line from your PR; see https://github.com/filecoin-station/spark-api/blob/67c5549d3f2381cdc7f155030212d23a6d58db61/Dockerfile#L12-L16

I am going to apply your suggestion for consistency, though.

---

Ah got it! I guess it's more consistent with the other project but less consistent with this file. To me file consistency > project consistency > ecosystem consistency. Otherwise we're sending signals that confuse poor simple-minded devs like me :D

---

>To me file consistency > project consistency > ecosystem consistency. Otherwise we're sending signals that confuse poor simple-minded devs like me :D

Sure, I agree.

I quickly copy&pasted the line from the other Dockerfile and did not review the surrounding lines/the diff. 🤷🏻